public/sw.js

```diff
@@ -1,20 +1,19 @@
 self.addEventListener('push',function (e){
-    var options = {
-        body: 'This notification was generated from a push!',
-        icon: 'images/example.png',
+  const options = {
+        body: 'New heart notification!',
```

Comment:
I would change this message to something more elegant like 'New message from your partner'